ffs/interfaces.go

```diff
@@ -16,7 +16,31 @@ type WalletManager interface {
 	// Balance returns the current balance for an address.
 	Balance(context.Context, string) (uint64, error)
 	// SendFil sends fil from one address to another
-	SendFil(ctx context.Context, from string, to string, amount *big.Int) error
+	SendFil(context.Context, string, string, *big.Int) error
+}
+
+type createVoucherConfig struct {
+	lane uint64
+}
+
+// CreateVoucherOption modifies a createVoucherCOnfig
+type CreateVoucherOption func(*createVoucherConfig)
+
+// WithCreateVoucherLane allows you to specify a non-0 lane
+func WithCreateVoucherLane(lane uint64) CreateVoucherOption {
+	return func(conf *createVoucherConfig) {
+		conf.lane = lane
+	}
+}
```

Comment:
Can this be moved to `ffs/api/options.go`?

Reply:
I actually removed `CreateVoucher()` from the interface, so these are no longer needed. May have to add it back in some day.